benchmark/single-source/DictionaryKeysContains.swift

```diff
@@ -37,17 +37,22 @@ public let DictionaryKeysContains = [
     runFunction: run_DictionaryKeysContains,
     tags: [.validation, .api, .Dictionary],
     setUpFunction: setup_DictionaryKeysContainsNative,
-    tearDownFunction: teardown_DictionaryKeysContains),
+    tearDownFunction: teardown_DictionaryKeysContains,
+    unsupportedPlatforms: [.linux]),
 ]
 #endif
 
 private var dictionary: [NSString: NSString]!
 
 private func setup_DictionaryKeysContainsNative() {
+#if os(Linux)
```

Comment:
These will be supported when https://github.com/apple/swift/pull/16736 lands.

Reply:
I figured.